owncloudApp/src/main/java/com/owncloud/android/presentation/ui/authentication/LoginActivity.kt

```diff
@@ -56,6 +56,8 @@ import com.owncloud.android.extensions.parseError
 import com.owncloud.android.extensions.showErrorInToast
 import com.owncloud.android.lib.common.accounts.AccountTypeUtils
 import com.owncloud.android.lib.common.accounts.AccountUtils
+import com.owncloud.android.lib.common.accounts.AccountUtils.Constants.OAUTH2_OIDC_SCOPE
+import com.owncloud.android.lib.common.authentication.oauth.OAuthConnectionBuilder
```

Comment:
The idea of new arch is to remove dependencies between app and library module. Why are you adding constants in the library that are only used at app module?

Reply:
That's is the ideal scenario but since the AppAuth library is tied to the presentation layer since it takes care of all the OAuth process, including the UI stuff... I need these dependencies here...

You might have thought of moving those dependencies (constants and methods) to the presentation layer but I'd rather include stuff like http urls, ssl, trustmanager as least as possible in the presentation layer

Reply:
I understand it, but i think that as app auth is an exception, its constants and methods should be included in app project. That way, we will avoid creating a constant in a submodule that will be only used in app project.

Reply:
I could do it with OAuthConnectionBuilder but not with `OAUTH2_OIDC_SCOPE` since `KEY_OAUTH2_SCOPE` is used in data layer.

Reply:
By the way, we would need to recheck more dependencies like the ones used in the code below

```
userAccount?.let {
    AccountUtils.getUsernameForAccount(it)?.let { username ->
        account_username.setText(username)
    }
}
```

Reply:
If oauth is not handled by library, i would move oauth stuff to data layer. What do you think?

Reply:
Yep, let's move the constants I need to use in this PR but we still have other ones related to OAuth in AccountUtils, that are being used during synchronization, we will check it soon.